spec/ruby/core/enumerable/drop_while_spec.rb

```diff
@@ -44,4 +44,9 @@
     multi = EnumerableSpecs::YieldsMulti.new
     multi.drop_while {|e| e != [6, 7, 8, 9] }.should == [[6, 7, 8, 9]]
   end
+
+  it "returns the nil as size when no block is given" do
```

Comment:
Got caught, hehehe. I think it could also give a nice a movie title... 'The Nil' :smile: